src/content/contributing/style-guide/index.md

```diff
@@ -16,11 +18,27 @@ Anyone! Ethereum.org is entirely open source, and many of its best pages are sub
 
 ## Audience {#audience}
 
-Ethereum.org content caters to a large and varied audience. Because ethereum.org caters to a wide spectrum of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
+- 60% of our website visitors do not own any Ether according to our survey responses
```

Comment:
Made a separate note about "ether" capitalization... per our current style guide we use lowercase, but as I noted, the Yellow Paper consistently capitalizes this, so we may want to separately consider switching the casing here.

But in the meantime:
```suggestion
- 60% of our website visitors do not own any ether according to our survey responses
```